tools/agent-ctl/src/main.rs

```diff
@@ -169,10 +171,11 @@ fn connect(name: &str, global_args: clap::ArgMatches) -> Result<()> {
         None => 0,
     };
 
-    let hybrid_vsock_port: u64 = args
+    let hybrid_vsock_port = args
         .value_of("hybrid-vsock-port")
         .ok_or("Need Hybrid VSOCK port number")
-        .map(|p| p.parse::<u64>().unwrap())
+        .map_err(|e| anyhow!(e))?
```

Comment:
Again you can collapse `.ok_or().map_err()` into just `.ok_or_else()` (or, theoretically to just `.ok_or()`, but `.ok_or_else()` is preferred, since it avoids constructing the error object on the happy path).